lib/MiLight/RgbwPacketFormatter.cpp

```diff
@@ -86,10 +86,23 @@ void RgbwPacketFormatter::updateColorWhite() {
 }
 
 void RgbwPacketFormatter::enableNightMode() {
-  uint8_t button = STATUS_COMMAND(ON, groupId);
-
-  command(button, 0);
-  command(button | 0x10, 0);
```

Comment:
I think perhaps what was there before almost gives you what you want.  It looks like the commands should actually be `STATUS_COMMAND(OFF, groupId) | 0x10`.  This would avoid needing to case each of the groups, and needing separate command IDs for night mode.

Reply:
Hmmm good idea... Will give this a try later...

Reply:
OK... This one works quite well:

uint8_t button = STATUS_COMMAND(OFF, groupId);
command(button | 0x10, 0);

Same as with my solution: Works when using udp gateway and REST API, but must be fired twice when using the REST API and the bulb is on.